tests.md

```diff
@@ -21,24 +21,36 @@ Refreshing the URL will re-run the tests via PhantomJS and in the browser.
 
 ## Selenium tests
 
-There is a bit of setup involved before you are able to run the Nightwatch selenium tests.
+To run selenium locally you need docker installed. Selenium runs in a docker container and connects to a CouchDB instance in another container. To start them run the command:
 
-First, in your CouchDB admin accounts, add a user:
+    npm run docker:up
 
-> user: tester  
-password: testerpass  
+You can run `npm run docker:logs` to see when the CouchDB instance is ready to go. You also need a Fauxton server instance runnning:
 
-Then on the command line:  
+    npm run dev
```

Comment:
i just tried that, `npm run dev` does not work for selenium runs, as the templates / htmls will throw a 404 because of a hack we introduced for webpack and `with` in underscore templates